var/spack/repos/builtin/packages/ripgrep/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ripgrep(Package):
+    """ripgrep is a line-oriented search tool that recursively searches
+    your current directory for a regex pattern.  ripgrep is similar to
+    other popular search tools like The Silver Searcher, ack and grep.
+    """
+
+    homepage = "https://github.com/BurntSushi/ripgrep"
+    url      = "https://github.com/BurntSushi/ripgrep/archive/11.0.2.tar.gz"
+
```

Comment:
hi, do you want to list yourself as package maintainer, @hartzell ?
```suggestion
    maintainers = ['hartzell']

```

Reply:
Not unless you insist, I'm between gigs at the moment and am not sure how OSS work I'll be able to do depending on where I land.  AKA, commitment issues....

Reply:
FWIW, I threw all three of these things in because a) I use them and b) I'm playing with the support for "stacking" spack installations and needed some things that were built with rust so that I can ensure that Lmod's compiler based hierarchies don't get confused (e.g. at one point D things like sambamba ended up in separate parts of the hierarchy because D->LLVM and therefore couldn't be loaded alongside things built with gcc)